codacy/values.yaml

```diff
@@ -358,51 +363,10 @@ minio:
 
 worker-manager:
   create: true
-  replicaCount: 1
   fullnameOverride: codacy-worker-manager
   config:
     play:
-      configFile: enterprise.conf
       logFile: logback.xml
     cloudAmqp:
```

Comment:
This could be empty, because of this if: https://bitbucket.org/qamine/worker-manager/src/f1c25d83f602660e095c49d923b082182c1f9a95/.helm/worker-manager/templates/configmap.yaml?at=dev#configmap.yaml-16:17

Maybe put an empty object here and a comment saying that it will use the one from global by default